andela/workdey
# Ft easy connection with taskees #123022021

# [#123022021] Easy connection with taskees
#### What does this PR do?

This PR implements the functionality that enables taskers to view taskees and their skillsets.
It also includes the functionality to view a taskee location
#### Description of Task to be completed?

A tasker can now view all taskees on the workdey platform and then know their skillsets at first glance. He/She may also decide to view the taskee location if he so wishes.
#### How should this be manually tested?
- Login as a tasker on the workdey website
- On the sidebar, you will see a button that reads Taskee skillsets
- Click this button and you will be taking to a page where you can see the taskees information
#### What are the relevant pivotal tracker stories?
#123022021
#### Screenshots (if appropriate)

![screen shot 2016-08-04 at 5 42 09 pm](https://cloud.githubusercontent.com/assets/14159416/17410304/f81ecd46-5a6a-11e6-802f-5408da992a3b.png)

![screen shot 2016-08-04 at 5 42 26 pm](https://cloud.githubusercontent.com/assets/14159416/17410386/5596fffc-5a6b-11e6-8a34-12ef85811d8e.png)
#### Questions:
